src/main/java/codeu/controller/RocketBotHandler.java

```diff
@@ -0,0 +1,60 @@
+package codeu.controller;
```

Comment:
I would split this into a separate pull request. Keep the implementation of a single bot separate from the proposed interface that all other bots should implement